simple/tests/stats/test_data/runner/config/config_driven.json

```diff
@@ -10,6 +10,12 @@
       "importType": "observations",
       "entityType": "Country",
```

Comment:
should this one also have a "format" field?

Reply:
No. This has the default variable per column format.

Reply:
I feel it's more clear to always set the format.

Reply:
SG - added.